pycaret/internal/plots/time_series.py

```diff
@@ -529,9 +499,7 @@ def plot_acf(
             fig=fig, fig_kwargs=fig_kwargs, fig_defaults=fig_defaults
         )
 
-    return_data_dict = {
-        "acf": corr_array,
-    }
+    return_data_dict = {"acf": acf(data, alpha=0.05, nlags=nlags)}
```

Comment:
You should reuse the `acf_data` returned from the subplot above here instead of calling acf function again.